integration/specs/tabset-1.spec.js

```diff
@@ -0,0 +1,35 @@
+const PageTabset = require('./../../src/components/Tabset/pageObject');
+
+const TABSET = '#tabset-1';
+
+const LEFT_ARROW = 'Left arrow';
+
+describe('Tabset with href example', () => {
+    beforeEach(() => {
+        browser.url('/#!/Tabset/1');
+        browser.refresh();
+    });
+
+    it('should select the first tab when selected', () => {
+        const tabset = new PageTabset(TABSET);
+        const tabItem = tabset.getItem(0);
+        tabItem.click();
+        expect(tabItem.isSelected()).toBe(true);
+    });
+    it('should select the secound tab when the first has focus and down key is pressed', () => {
```

Comment:
second ( but to be more precise put last tab because it will select the last tab, not always de second is the last ) and left arrow key